ner_v2/detectors/numeral/number/number_detection.py

```diff
@@ -81,7 +81,7 @@ def get_supported_languages():
         cwd = os.path.dirname(os.path.abspath(__file__))
         cwd_dirs = [x for x in os.listdir(cwd) if os.path.isdir(os.path.join(cwd, x))]
         for _dir in cwd_dirs:
-            if len(_dir.rstrip(os.sep)) == 2:
+            if len(_dir.rstrip(os.sep)) in [2, 5]:
```

Comment:
its for checking the supported languages, the languages for which folders are created. since the name folder name is dependent on the language and language for Chinese traditional is 'zh-TW' and other languages are only described in 2 chars like english 'en' and Hindi 'hi' or  Bengali as 'bn'
So for Chinese traditional to be eligible in supported languages need to add this.

Reply:
Understood, thanks.